package.json

```diff
@@ -19,9 +19,10 @@
     "chalk": "^0.5.1",
     "debug": "^2.1.0",
     "event-stream": "^3.1.7",
-    "gulp-util": "^3.0.2",
+    "fancy-log": "^1.3.2",
     "lodash.assign": "^3.0.0",
-    "mini-lr": "^0.1.8"
+    "mini-lr": "^0.1.8",
+    "vinyl": "^2.1.0"
```

Comment:
Is `vinyl` used outside of `test/index.js`? I may be missing something but i think that `vinyl` should be in the `devDependencies`.

Reply:
Ping @zbennett10